dipy/reconst/cti.py

```diff
@@ -0,0 +1,467 @@
+                          + D[..., 1, 1] ** 2 + C[..., 2]
+                          + D[..., 2, 2]**2 - C[..., 5]
+                          - D[..., 0, 0] * D[..., 1, 1] -
+                          C[..., 4] - D[..., 0, 0] * D[..., 2, 2]
+                          - C[..., 3] - D[..., 1, 1] * D[..., 2, 2]
+                          + 3 * (C[..., 17] + D[..., 0, 1] ** 2 + C[..., 16]
+                                 + D[..., 0, 2] ** 2
+                                 + C[..., 15] + D[..., 1, 2] ** 2))
+        mean_D = np.trace(D) / 3
+        K_aniso = (6/5) * (Variance / (mean_D ** 2))
+        return K_aniso
+
+    @property
+    def K_iso(self):
+        r""" Returns the isotropic Source of Kurtosis (K_iso)
+
+        Notes
+        -----
+        The K_iso is defined as :
+
+        :math::
+            \[K_{iso} = 3 \cdot \frac{V({\overline{D}^c})}{\overline{D}^2}\]
+
+        where: \(K_{iso}\) is the isotropic kurtosis,
+            \(V({\overline{D}^c})\) represents the variance of the diffusion
+            tensor raised to the power c,
+            \(\overline{D}\) is the mean of the diffusion tensor.
+
+        """
+        C = self.ct
+        mean_D = self.md
+        Variance = 1/9 * (C[..., 0] + C[..., 1] + C[..., 2] + 2 * C[..., 5]
+                          + 2 * C[..., 4] + 2 * C[..., 3])
+        K_iso = 3 * (Variance / (mean_D ** 2))
+        return K_iso
+
+    @auto_attr
+    def K_total(self):
+        r""" Returns the total execess Kurtosis. (K_total)
+
+            Notes
+            -----
+            The K_total is defined as :
+
+            :math::
+                \[\Psi = \frac{2}{5} \cdot \frac{D_{11}^2 + D_{22}^2 + D_{33}^2
+                                                 + 2D_{12}^2 + 2D_{13}^2 +
+                                                 2D_{23}^2{\overline{D}^2} -
+                                                 \frac{6}{5} \]
+                \[{\overline{W}} = \frac{1}{5} \cdot (W_{1111} + W_{2222}
+                                                      + W_{3333} + 2W_{1122}
+                                                      + 2W_{1133} + 2W_{2233})\
+                  ]
+
+            where \(\Psi\) is a variable representing a part of the total
+            excess kurtosis,
+            \(D_{ij}\) are elements of the diffusion tensor,
+            \(\overline{D}\) is the mean of the diffusion tensor.
+            \{\overline{W}} is the mean kurtosis,
+            \(W_{ijkl}\) are elements of the kurtosis tensor.
+        """
+
+        mean_K = self.mkt()
+        D = self.quadratic_form
+        mean_D = np.trace(D) / 3
+        psi = 2 / 5 * ((np.sqrt(D[..., 0, 0]) + np.sqrt(D[..., 1, 1])
+                        + np.sqrt(D[..., 2, 2])
+                        + 2 * np.sqrt(D[..., 0, 1]) + 2 * np.sqrt(D[..., 0, 2])
+                        + np.sqrt(D[..., 1, 2])) / (mean_D ** 2)) - (6/5)
+
+        excess_K = 1/5 * mean_K + psi
```

Comment:
In addition to the mistake in the calculation in psi. Why are you multiplying mean_K by 1/5? See equation 9 of the CTI neuroimage 2020 paper. According to that Ktotal = mean_K + psi. Avoid also using the variable name excess_K since it may be misleading. I would just combine the mean_K and psi directly in the function return, i.e.:

return mean_K + psi